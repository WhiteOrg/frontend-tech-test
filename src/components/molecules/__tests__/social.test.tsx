import React from "react";
import renderer from "react-test-renderer";
import Social from "../social";

describe("MOLECULES: Render testing", () => {
  test("Should render Social", () => {
    const component = renderer.create(
      <div>
        <Social>Share this Tournament</Social>
      </div>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
