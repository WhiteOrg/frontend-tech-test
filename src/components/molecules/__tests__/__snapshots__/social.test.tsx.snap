// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATOMS: Render testing Should render Hr 1`] = `
<div>
  <div
    className="sc-AxiKw gIfVsQ"
  >
    <p
      className="sc-AxjAm iSHejg"
      color="#f9cb11"
      size="medium"
    >
      Share this Tournament
    </p>
    <div
      className="sc-AxhCb gewXQy"
    >
      <img
        className="sc-AxirZ kOBdo"
        src="twitter.svg"
      />
      <img
        className="sc-AxirZ kOBdo"
        src="facebook.svg"
      />
    </div>
  </div>
</div>
`;

exports[`ATOMS: Render testing Should render Social 1`] = `
<div>
  <div
    className="sc-AxiKw gIfVsQ"
  >
    <p
      className="sc-AxjAm iSHejg"
      color="#f9cb11"
      size="medium"
    >
      Share this Tournament
    </p>
    <div
      className="sc-AxhCb gewXQy"
    >
      <img
        className="sc-AxirZ kOBdo"
        src="twitter.svg"
      />
      <img
        className="sc-AxirZ kOBdo"
        src="facebook.svg"
      />
    </div>
  </div>
</div>
`;

exports[`MOLECULES: Render testing Should render Social 1`] = `
<div>
  <div
    className="sc-AxiKw gIfVsQ"
  >
    <p
      className="sc-AxjAm iSHejg"
      color="#f9cb11"
      size="medium"
    >
      Share this Tournament
    </p>
    <div
      className="sc-AxhCb gewXQy"
    >
      <img
        className="sc-AxirZ kOBdo"
        src="twitter.svg"
      />
      <img
        className="sc-AxirZ kOBdo"
        src="facebook.svg"
      />
    </div>
  </div>
</div>
`;
